persistence/secrets.go

```diff
@@ -0,0 +1,78 @@
+package persistence
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/boltdb/bolt"
+	"github.com/golang/glog"
+	"github.com/open-horizon/anax/containermessage"
+)
+
+const SECRETS = "secrets"
+
+func secretKey(agId string, svcName string) string {
+	return fmt.Sprintf("%s/%s", agId, svcName)
+}
+
+func SaveSecrets(db *bolt.DB, secrets map[string]containermessage.Secret, agId string, svcName string) error {
```

Comment:
You will need an update secret function.
Also you'll need to consider the state that the agent's /secrets API will need in order to know which services a secret has been delivered to.
Finally, I'm not sure that agid is the right key, I think it will have to be service id/name because of the state that the /secret neds to maintain, and it is based on a service not an agreement id.

Reply:
Addressed these changes